torch/distributed/tensor/_ops/_pointwise_ops.py

```diff
@@ -680,11 +701,13 @@ def list_pointwise_strategy(
         OpStrategy: generated strategy
     """
 
-    def args_tuple_strategies(args_schema: tuple[object, ...]) -> list[TupleStrategy]:
+    def args_tuple_strategies(
+        args_schema: tuple[object, ...],
```

Comment:
```suggestion
        args_schema: tuple[TupleStrategy, object, ...],
```
Seems like we require the first arg to be a tuple strategy, so might as well make the typing expect that too.